components/dashboard/src/App.tsx

```diff
@@ -345,10 +368,49 @@ function App() {
         false,
     );
 
+    const acceptAllCookies = () => {
```

Comment:
If we introduced a `ConsentCookieContext` I think all of these methods could move into the modal/banner/consent-cookie.ts - not sure though. :thinking: 
I suggest we sync next week and play around with it together. :keyboard:  :pray: